Add documentation for audit trails token (#644)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx

# API Tokens

> HCP Terraform supports four distinct types of API tokens with varying levels of access: user, team, organization, and audit trails. There are differences in access levels and generation workflows for each of these token types, which are outlined below.

Reviewer: nit: I wonder if it's time to not count it exactly? 

```suggestion
HCP Terraform supports many distinct types of API tokens with varying levels of access: user, team, organization, and audit trails. There are differences in access levels and generation workflows for each of these token types, which are outlined below.
```

-> **Note:** This endpoint cannot be accessed with [organization tokens](#organization-api-tokens). You must access it with a [user token](/terraform/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](#team-api-tokens).

> ## Audit Trails Tokens

Reviewer 2: To make the plural-singular agree here, we need to either have "trails" or "tokens" be plural, so I suggest keeping "trails" singular:
```suggestion
## Audit trail tokens
```

> Tokens may be generated for reading organization audit trails. They are intended to be used in the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.

Reviewer: ```suggestion
Tokens may be generated for reading organization audit trails, for example by the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```

Reviewer 2: Style suggestion for active voice and expanding a bit to make this a bit more generic for other metrics integrations:
```suggestion
You can generate an audit trails token to read an organization's [audit trails](/terraform/cloud-docs/api-docs/audit-trails). Use this token type to authenticate integrations pulling audit trail data, for example, using the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```

> To manage the token for reading audit trails for a specific organization, go to **Organization settings > API Token** and use the controls under the "Audit Token" header.

Reviewer 2: ```suggestion
To manage an organization's audit trails token, go to **Organization settings > API Token** and use the settings under the "Audit Token" header.
```

Reviewer 2: Is the header for "Audit Token" plural or singular in the UI?

Author: It's singular!

> Each organization can have **one** valid audit trails token at a time. Only [organization owners](/terraform/cloud-docs/users-teams-organizations/teams#the-owners-team) can generate or revoke an organization's audit trails token.

Reviewer 2: We only use bold for UI elements
```suggestion
Each organization can only have a _single_ valid audit trails token. Only [organization owners](/terraform/cloud-docs/users-teams-organizations/teams#the-owners-team) can generate or revoke an organization's audit trails token.
```
